### PR TITLE
[JS v7] Add Prisma ORM Integration Docs.

### PR DESCRIPTION
**Pull request by onurtemizkan**
- Added a simple example about the use of new Prisma integration. (https://github.com/getsentry/sentry-javascript/pull/4931)

### CODE REVIEW

File: src/platforms/node/common/performance/database/opt-in.mdx [hunk 1]
```diff
@@ -0,0 +1,25 @@
+---
+title: Opt-in
+---
+
+## Prisma ORM Integration _(Available from `7.0.0`)_
+
+Sentry supports tracing [Prisma ORM](https://www.prisma.io/) fetchers with Prisma Integration. This integration is an opt-in feature and requires a `PrismaClient` instance provided.
```
Comment (AbhiPrasad) [inline src/platforms/node/common/performance/database/opt-in.mdx:7]: Can we add what kind of spans will be generated?

File: src/platforms/node/common/performance/database/index.mdx [hunk 2]
```diff
@@ -0,0 +1,23 @@
+---
+title: Database integrations
```
Comment (imatwawana) [inline src/platforms/node/common/performance/database/index.mdx:2]:
```suggestion
title: Database Integrations
```

File: src/platforms/node/common/performance/database/index.mdx [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+---
+
+Node.js integrations support tracking database queries as spans.
+
+## Supported platforms
```
Comment (imatwawana) [inline src/platforms/node/common/performance/database/index.mdx:7]:
```suggestion
## Supported Patforms
```

File: src/platforms/node/common/performance/database/opt-in.mdx (hunk 1 above)
Comment (imatwawana) [inline src/platforms/node/common/performance/database/opt-in.mdx:7]:
```suggestion
Sentry supports tracing [Prisma ORM](https://www.prisma.io/) fetchers with Prisma integration. This integration is an opt-in feature and requires that a `PrismaClient` instance is provided.
```

**Review by imatwawana — COMMENTED** [review]
Made some style and wording edits

**Comment by onurtemizkan** [comment]
Had to convert this PR to a larger scope (v7 opt-in db integrations Apollo + GQL + Prisma), to avoid confusion / potential conflicts when we're merging for v7. Hope that's fine. @AbhiPrasad, @imatwawana.

**Review by imatwawana — APPROVED** [review]
A few wording edits + I made a couple fixes I missed the first time around, but this mostly looks good to me!

**Review by AbhiPrasad — APPROVED** [review]

**Pull request merged**